Services/DataX.Config/DataX.Config/ConfigGeneration/Processor/S500_ResolveOutputs.cs

```diff
@@ -196,8 +197,9 @@ private async Task<FlowBlobOutputSpec> ProcessOutputBlob(string configName, Flow
                 var accountName = ParseBlobAccountName(connectionString);
                 var blobPath = $"wasbs://{uiOutput.Properties.ContainerName}@{accountName}.blob.core.windows.net/{uiOutput.Properties.BlobPrefix}/%1$tY/%1$tm/%1$td/%1$tH/${{quarterBucket}}/${{minuteBucket}}";
                 var secretId = $"{configName}-output";
-                var blobPathSecret = await KeyVaultClient.SaveSecretAsync(sparkKeyVaultName, secretId, blobPath, true);
-                await KeyVaultClient.SaveSecretAsync(sparkKeyVaultName, $"datax-sa-{accountName}", ParseBlobAccountKey(connectionString), false);
+                var uriPrefix = (Configuration[Constants.ConfigSettingName_SparkType].Length > 0 && Configuration[Constants.ConfigSettingName_SparkType] == Constants.SparkTypeDataBricks) ? Constants.PrefixSecretScope : Constants.PrefixKeyVault;
+                var blobPathSecret = await KeyVaultClient.SaveSecretAsync(sparkKeyVaultName, secretId, blobPath, uriPrefix, true);
+                await KeyVaultClient.SaveSecretAsync(sparkKeyVaultName, $"datax-sa-{accountName}", ParseBlobAccountKey(connectionString), uriPrefix, false);
```

Comment:
can you refactor datax-sa- to a constant

Reply:
Done